app/models/ticket.rb

```diff
@@ -121,7 +125,15 @@ def status_times
     total
   end
 
+  def time_consumed_as_duration
+    Duration.new(time_consumed*60)
+  end
+
   protected
+    def set_time_consumed
+      self.time_consumed = Duration.new(:days => self.consumed_days.to_i, :hours => self.consumed_hours.to_i, :minutes => self.consumed_minutes.to_i).total/60 if self.consumed_days || self.consumed_hours || self.consumed_minutes
```

Comment:
Line is too long. [228/80]<br>Use the new Ruby 1.9 hash syntax.<br>Redundant `self` detected.<br>Surrounding space missing for operator '/'.